### ISSUE TITLE
Rephrase newer version warning

### ISSUE THREAD

**Issue by peterdesmet** [gbif/ipt#2200]
The not-latest version of a dataset show the following warning:

<img width="736" alt="Screenshot 2023-11-16 at 08 51 53" src="https://github.com/gbif/ipt/assets/600993/f55d1490-ffdd-4ce1-99c9-6d715dae84c1">

I would rephrase that (no exclamation mark) and link directly to the latest version (i.e. resource URL without `&v=`:):

> There is a [newer version](https://ipt3.gbif-uat.org/resource?r=camtrap-dp-example) of the record available.

**Comment by mike-podolskiy90** [comment]
Thanks Peter. Good point

**Comment by peterdesmet** [comment]
Gs this been deployed to UAT?

**Comment by mike-podolskiy90** [comment]
Yes it is

**Comment by peterdesmet** [comment]
Aha, I see it now at https://ipt3.gbif-uat.org/resource?r=camtrap-dp-example&v=6.0:

<img width="605" alt="Screenshot 2023-11-21 at 15 22 14" src="https://github.com/gbif/ipt/assets/600993/bbbb9b13-054c-4ba2-ab89-26fdbf3ca6b1">

I notice that the link (= `newer version`) has the same colour as the text (`--alert-warning-text-color`), making it hard to see this is a link. Maybe the `alert-link` class should be used to make it stand out: https://getbootstrap.com/docs/5.0/components/alerts/#link-color

This applies to all warnings.
